src/styles/header.scss

```diff
@@ -0,0 +1,64 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  padding: 0 50px;
+  background-color: #fff;
+}
+
+.logo__img {
+  display: block;
+  width: 40px;
+  height: 40px;
+}
+
+.nav {
+  &__list {
+    display: flex;
+    margin: 0;
+    padding: 0;
+
+    list-style: none;
+  }
+
+  &__item {
+    width: fit-content;
+    margin-right: 20px;
+
+    &:last-child {
+      margin-right: 0;
+    }
+  }
+
+  &__link {
+    text-transform: uppercase;
+    font-weight: 500;
+    font-size: 12px;
+    display: block;
+    line-height: 60px;
+    color: #060b35;
+    text-decoration: none;
+    transition: all 300ms linear;
+
+    &:hover {
+      color: #00acdc;
+    }
+  }
+}
+
+a.is-active {
+  position: relative;
+  color: #00acdc;
+}
+
+.is-active::after {
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  width: 100%;
+  height: 4px;
+  background-color: #00acdc;
+  border-radius: 8px;
+  content: '';
+}
```

Comment:
The same applies here. It's better to use the class name 'nav__link--active' instead of 'is-active'. This way, the code will be more consistent and it will be easier to understand that this style is related to the active state of the nav link.